tests/unit/sortable/core.js

```diff
@@ -42,4 +42,57 @@ QUnit.test( "ui-sortable-handle applied to appropriate element", function( asser
 	assert.equal( el.find( ".ui-sortable-handle" ).length, 0, "class name removed on destroy" );
 } );
 
+// gh-1998
+QUnit.test( "drag & drop works with a zero-height container", function( assert ) {
+	var ready = assert.async();
+	assert.expect( 3 );
+
+	var el = $( "<ul class='list-gh-1998'>\n" +
+		"	<style>" +
+		"		.list-gh-1998 {\n" +
+		"			margin: 0;\n" +
+		"			padding: 0;\n" +
+		"		}\n" +
+		"		.list-item-gh-1998 {\n" +
+		"			float: left;\n" +
+		"			display: block;\n" +
+		"			width: 100px;\n" +
+		"			height: 100px;\n" +
+		"		}\n" +
+		"	</style>\n" +
+		"	<li class='list-item-gh-1998'>Item 1</li>\n" +
+		"	<li class='list-item-gh-1998'>Item 2</li>\n" +
+		"	<li class='list-item-gh-1998'>Item 3</li>\n" +
+		"</ul>" )
+		.sortable()
+		.appendTo( "#qunit-fixture" );
+
+	function step1() {
+		el.find( "li" ).eq( 0 ).simulate( "drag", {
+			dx: 100,
+			dy: 3,
```

Comment:
@fnagel I'm not sure why `dy` matters here since we're dragging horizontally but it had a visible effect on the test... I'm not sure if it's an actual issue or just a jQuery Simulate quirk but in manual testing it seemed to work fine so I gave up on debugging this.

Reply:
Sorry, I'm not really helpful here, but I guess you are right: could be a jQuery Simulate quirk too.
Having some tests should be fine...